runtime/moonbeam/src/lib.rs

```diff
@@ -1524,6 +1524,7 @@ pub type SignedExtra = (
 	frame_system::CheckNonce<Runtime>,
 	frame_system::CheckWeight<Runtime>,
 	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
+	frame_metadata_hash_extension::CheckMetadataHash<Runtime>,
 	cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim<Runtime>,
```

Comment:
Just asking, is the order important here? In the guide I see `CheckMetadataHash` is specified after `StorageWeightReclaim`.

Reply:
From what I check, `CheckMetadataHash` does not have side-effects on `StorageWeightReclaim`.

Also, they have this in the documentation:
```md
For maximum efficiency, make sure that `StorageWeightReclaim` is last in the list.
```